### PR TITLE
BW-1305 Adds GET /runs endpoint

### PR DESCRIPTION
**Pull request by salonishah11**
Closes https://broadworkbench.atlassian.net/browse/BW-1305

### CODE REVIEW

File: service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java [hunk 1]
```diff
@@ -22,28 +24,65 @@ public class RunsApiController implements RunsApi {
...
+
+  private Date convertToDate(OffsetDateTime submissionTimestamp) {
+    if (submissionTimestamp != null) {
+      return new Date(submissionTimestamp.toInstant().toEpochMilli());
+    }
+
+    return null;
+  }
+
+  private LogRunRequest convertToLogRunRequest(WorkflowQueryResult queryResult) {
```
Comment (cjllanwarne) [inline service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:48]: This feels more like a `LogRunResponse` than a request?

File: service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java [hunk 2]
```diff
@@ -75,17 +112,11 @@ public ResponseEntity<RunStateResponse> postRun(String workflowUrl, Object workf
...
-              new RunStateResponse().runId(runId).state(RunState.QUEUED), HttpStatus.CREATED);
+      return new ResponseEntity<>(
+          new RunStateResponse().runId(runId).state(RunState.QUEUED), HttpStatus.CREATED);
     } catch (cromwell.client.ApiException e) {
-      System.out.println(e);
-      result =
-          new ResponseEntity<>(
-              new RunStateResponse().runId(runId).state(RunState.SYSTEM_ERROR),
-              HttpStatus.INTERNAL_SERVER_ERROR);
+      System.out.println(e.getMessage());
```
Comment (cjllanwarne) [inline service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:118]: We should log the full error even if we only return the short message to the user

**Review by cjllanwarne — APPROVED** [review]
LGTM, a few TOL level comments

File: service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java (hunk 1 above)
Comment (salonishah11) [reply service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:48]: Good point. I will rename it.

**Review by kpierre13 — APPROVED** [review]

**Pull request merged**